neo/app/views/index.scala.html

```diff
@@ -9,15 +9,19 @@
     <div id="logo">
     </div>
 
-    <div id='how-to'>
-      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info btn-sm">How to use this demo</a>  
+    <div id='video-demo'>
+      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info btn-xs">How to use this demo</a>  
```

Comment:
Agreed.  Also it will be good to move this link on the right of the page, close to "Powered by Cloudberry".

In addition, can you add a screenshot of the new UI to this PR to make the effect more obvious?